src/openrct2/peep/Peep.h

```diff
@@ -368,10 +368,10 @@ enum PEEP_FLAGS : uint32_t
 
 enum PEEP_NEXT_FLAGS
 {
-    PEEP_NEXT_FLAG_DIRECTION_MASK = 0x3,
-    PEEP_NEXT_FLAG_IS_SLOPED = (1 << 2),
-    PEEP_NEXT_FLAG_IS_SURFACE = (1 << 3),
-    PEEP_NEXT_FLAG_UNUSED = (1 << 4),
+    PEEP_NEXT_FLAG_DIRECTION_MASK = 0b00000011,
+    PEEP_NEXT_FLAG_IS_SLOPED = 0b00000100,
```

Comment:
I appreciate the mask `PEEP_NEXT_FLAG_DIRECTION_MASK` now being in bitwise notation — I agree it increases legibility.

However, I do think the three flags should retain the bitshift notation — it makes it immediate obvious which bit it concerns, rather than having to count zeros.